### PR TITLE
Added missing placement_group field to linode_instances data source

### PR DESCRIPTION
**Pull request by ezilber-akamai**
## 📝 Description

The linode_instances data source now exposes the placement_group field on returned instances.

### Integration Tests

- `make PKG_NAME=linode/instance ARGS="-run TestAccDataSourceInstances_basic" int-test`
- `make PKG_NAME=linode/instance ARGS="-run TestAccDataSourceInstances_withPG" int-test`
- `make PKG_NAME=linode/instance ARGS="-run TestAccResourceInstance_withPG" int-test`
- `make PKG_NAME=linode/instance ARGS="-run TestAccResourceInstance_pgAssignment" int-test`

### CODE REVIEW

File: linode/instance/schema_datasource.go [hunk 1]
```diff
@@ -406,4 +408,38 @@ var instanceDataSourceSchema = map[string]*schema.Schema{
 			},
 		},
 	},
+	"placement_group": {
+		Type:     schema.TypeList,
+		Computed: true,
+		Optional: true,
```
Comment (yec-akamai) [inline linode/instance/schema_datasource.go:414]: We probably don't need to put Optional here, since it's only for read.

**Review by yec-akamai — COMMENTED** [review]
Overall looks good! Just a small thing to fix:

File: linode/instance/schema_datasource.go
Comment (yec-akamai) [inline linode/instance/schema_datasource.go:1]: Can we add the new attributes to the instance data source doc?

Comment (ezilber-akamai) [reply linode/instance/schema_datasource.go:1]: Done!

File: linode/instance/schema_datasource.go (hunk 1 above)
Comment (ezilber-akamai) [reply linode/instance/schema_datasource.go]: Fixed!

**Review by yec-akamai — APPROVED** [review]
Nice work! Tests passed locally.

**Review by ykim-1 — APPROVED** [review]
LGTM, all tests pass locally. Nice work!

**Pull request merged**